.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Bug Report
+about: You're experiencing an issue with the Consul Helm that is different than the documented behavior.
+title: "[BUG]"
```

Comment:
Do we want this to automatically be prepended to the title when people choose the template? Looking for reviewer comments on this

Reply:
sweet, done!